### DISCUSSION TITLE
We need documentation - on the technical angle, not 'how to'

### DISCUSSION THREAD

**Discussion by stephantual** [comfyanonymous/ComfyUI#2385] [Q&A]
I'd gladly pay money for an indepth tutorial on what every pararameter of every node does in depth. For example, how "Clip" works behind the scenes. There are some great GH thread, but am i the only one finding myself having to run a google search every 3 second? 

If it allready exist, would be greatful for any pointer, thank you.

**Comment by ltdrdata** [comment]
I recommend this documents. (You may need a translator)
This document is the most in-depth among the node description documents I've seen.
https://comfyui.creamlab.net/

For example,
https://comfyui.creamlab.net/guides/f_KSamplerAdvanced_denoise
https://comfyui.creamlab.net/nodes/CLIPTextEncode

**Reply by stephantual** [reply]
Thank you so much! I appreciate your help
